ObjectMapper/Core/Mapper.swift

```diff
@@ -412,38 +412,31 @@ extension Mapper where N: Hashable {
 }
 
 extension Dictionary {
-	internal func map<K: Hashable, V>(@noescape f: Element -> (K, V)) -> [K : V] {
```

Comment:
For such a low layer of abstraction, I don't think applying "reduce" everywhere is a good idea. Performance hurts.